docs/example-apps/rails.md

```diff
@@ -1,45 +1,30 @@
-
-### Custom Application Components
-
-#### [Dockerfile](https://github.com/convox-examples/rails/blob/master/Dockerfile)
-
-Starting from the [ruby:2.6.4](https://hub.docker.com/_/ruby/) image, the `Dockerfile` defines the steps necessary to turn the application code into an image that is ready to run. 
-
-This `Dockerfile` has 3 steps and they are executed in a particular order to take advantage of Docker's build caching behavior.
-
-1. `bundle install` and `yarn install` are run to install dependencies after copying just the files needed to run these commands. This will ensure that the output these commands are cached unless one of these files changes.
-
-2. The application source is copied over. These files will change frequently so this step of the build will very rarely be cached.
-
-3. Finally, after setting the appropriate environment variables the assets are precompiled.
-
-#### [convox.yml](https://github.com/convox-examples/rails/blob/master/convox.yml)
+### Preparing your application
 
-The `convox.yml` manifest explains how to run the application. The manifest for this application has two sections:
-
-1. Resources: These are network-attached dependencies of the application. In this application we have a single resource, a `postgres` database.
+To deploy your rails application in your local rack your need to make a few changes to it. 
 
-2. Services: These are the web-facing services of the application. This application has a single service named `web` which is built from the local directory.
+1. To run your application you will need a [Dockerfile]. Create a file in the root of your project with the name `Dockerfile` and the content of our [example](https://github.com/convox-examples/rails). Notice that ours is using ruby-3.0, if you need a different version feel free to change the version on the first line of the file.
+2. To tell Convox how it should understand your application, your project needs to have a [convox.yml](https://docs.convox.com/configuration/convox-yml). Create a file in the root of your project with the name `convox.yml` and the content of our [example](https://github.com/convox-examples/rails/blob/master/convox.yml).
+3. To avoid Docker from loading unnecessary files you need to define a [.dockerignore](https://docs.docker.com/engine/reference/builder/#dockerignore-file). Create a file in the root of your project with the name `.dockerignore` and the content of our [example](https://github.com/convox-examples/rails/blob/master/.dockerignore).
+4. If you are using Rails 6.0+ you need to add the following line inside `config/environments/development.rb` ```config.hosts << 'api.roadrunner.dev.convox'```
+5. In your gemfile add the following line `gem 'pg'`
+6. To use Convox' database resource, in your `config/database.yml` add the following line under `default` ```  url: <%= ENV['DATABASE_URL'] %>```. Your `database.yml` should look like [this](https://github.com/convox-examples/rails/blob/master/config/database.yml)
```

Comment:
We should really still reference where the env var comes from and why it's important.  If the customer has a different name for their resource in the convox.yml then this line won't be true.

Reply:
👍🏻